painttiming.bs

```diff
@@ -62,9 +89,28 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Contentful</dfn>: An [=element=] or a [=pseudo-element=] counts as contentful when one or more of the following apply:
+* The element has a [=text node=] child, with at least one character, excluding [=document white space characters=].
+* The element is a [=replaced image=] representing a [=loaded=] image.
+* The element's [=computed=] style contains a [=loaded=] [=background-image=].
+* The element is a [=canvas=] with its [=context mode=] set to any value other than <code>none</code>.
+* The element is an [=svg element with rendered descendants=].
+* The element's [=computed=] style contains a [=contentful=] and [=paintable=] [=generated content pseudo-element=].
+
+<dfn>Paintable</dfn>: An [=element=] or a [=generated content pseudo-element=] counts as paintable when all of the following apply:
```

Comment:
Why do we use `generated content pseudo-element` here but `pseudo-element` above? I think we should be consistent.

Reply:
gotcha